src/index.html

```diff
@@ -1,13 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.8/css/line.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
     <title>JS-Capstone</title>
-</head>
-<body>
-    <h2>Welcome to Capstone Project 2</h2>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="logo">Abel&Alijan TV</div>
+        <ul>
+          <li>Shows <span class="total">(6)</span></li>
+          <li>Episodes</li>
+          <li>More</li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- [ ] As per the mockup in the project requirements, your navigation bar is supposed to look similar to the mockup. At the moment your navigation items are aligned to the right which is diferent from the mockup. Kindly fix this.
- Given mockup
![Screen Shot 2023-06-02 at 2 31 58 PM](https://github.com/ProgramKingAbel/MICROVERSE-JS-CAPSTONE/assets/62849872/a2d54587-9ff0-4125-8e33-f9d70dcab817)

- Your design
![Screen Shot 2023-06-02 at 2 33 15 PM](https://github.com/ProgramKingAbel/MICROVERSE-JS-CAPSTONE/assets/62849872/25c3e56c-e2e7-472a-b29a-60b258ba3a7c)